src/api/unified/blas.cpp

```diff
@@ -10,6 +10,14 @@
 #include <af/blas.h>
 #include "symbol_manager.hpp"
 
+AFAPI af_err af_gemm(af_array *out,
+                     const af_mat_prop optLhs, const af_mat_prop optRhs,
+                     const void* alpha, const af_array lhs, const af_array rhs,
+                     const void* beta) {
+    CHECK_ARRAYS(out, lhs, rhs);
```

Comment:
Addressed this in cc43610. It failed before since `out` is an `af_array*`. Now we pass in `*out`, which doesn't make `CHECK_ARRAYS` fail because the `checkArray` function it eventually calls [returns true if the array is null](https://github.com/arrayfire/arrayfire/blob/ac9119d895c6fdb21640a0cd8e7cba4bf8fac923/src/api/unified/symbol_manager.cpp#L242).